httpapi_plugins/ftd.py

```diff
@@ -145,14 +181,18 @@ def logout(self):
         self.access_token = None
 
     def _send_auth_request(self, path, data, **kwargs):
+        error_msg_prefix = 'Server returned an error during authentication request'
+        return self._send_request(kwargs, path, error_msg_prefix, data=data)
+
+    def _send_request(self, kwargs, path, error_msg_prefix, data=None):
```

Comment:
Let's rename this method to avoid confusion with public `send_request`. As this method sends a request ignoring all HTTP errors, it can be smth like `_send_request_ignoring_retries` or smth...

Reply:
Also, the `kwargs` argument looks awkward. Why not to make it `**kwargs` at the end as usual?

Reply:
fixed issue with `kwargs`, changed the name to the `_send_service_request` cause the main difference with `send_request` method is that we using `_send_service_requests` for service or auxiliary requests.